prusa_slicer_post_processing_script.py

```diff
@@ -690,16 +711,16 @@ def verifyinfillpolys(self,minDistForValidation:float=0.5)->None:
                     continue
                 if poly.area<self.parameters.get("MinArea"):
                     if self.parameters.get("PrintDebugVerification"):print(f"Layer {self.layernumber}: Poly{idp} has to little area: {poly.area:.2f}")
-                    continue               
+                    continue
                 for ohp in overhangs:
                     if poly.distance(ohp)<minDistForValidation:
                         if ohp.length>self.parameters.get("MinBridgeLength"):
                             self.validpolys.append(poly)
                             self.deleteTheseInfills.append(idp)
                             break
-                if self.parameters.get("PrintDebugVerification"):print(f"Layer {self.layernumber}: Poly{idp} is not close enough to overhang perimeters")        
+                    if self.parameters.get("PrintDebugVerification"):print(f"Layer {self.layernumber}: Poly{idp} is not close enough to overhang perimeters")
```

Comment:
Changed indentation to print for every failed `ohp` instead of once every time after the loop.

Reply:
Everything below this point is whitespace removal.